File: back/src/main/java/csh/back/domain/vote/vote/dto/response/VoteConfirmResponse.java
package csh.back.domain.vote.vote.dto.response;

import csh.back.domain.vote.vote.enums.VoteConfirmStatus;

import java.util.List;

public record VoteConfirmResponse(
        String voteStatus,
        Long confirmedPlaceId,
        boolean isTie
        )
{
    public static VoteConfirmResponse of(String voteStatus, Long confirmedPlaceId, boolean isTie) {
       return new VoteConfirmResponse(
               voteStatus,
               confirmedPlaceId,
               isTie
       );
    }
}

File: back/src/main/kotlin/csh/back/domain/vote/vote/dto/response/VoteConfirmResponse.kt
package csh.back.domain.vote.vote.dto.response

data class VoteConfirmResponse(
    val voteStatus: String,
    val confirmedPlaceId: Long,
    val isTie: Boolean,
) {
    companion object {
        @JvmStatic
        fun of(voteStatus: String, confirmedPlaceId: Long, isTie: Boolean): VoteConfirmResponse =
            VoteConfirmResponse(voteStatus, confirmedPlaceId, isTie)
    }
}
